opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/entrypoint/pis/Xs2aGetPaymentStatusEntrypoint.java

```diff
@@ -0,0 +1,99 @@
+package de.adorsys.opba.protocol.xs2a.entrypoint.pis;
+
+import de.adorsys.opba.protocol.api.common.ProtocolAction;
+import de.adorsys.opba.protocol.api.dto.context.ServiceContext;
+import de.adorsys.opba.protocol.api.dto.request.payments.PaymentStatusBody;
+import de.adorsys.opba.protocol.api.dto.request.payments.PaymentStatusRequest;
+import de.adorsys.opba.protocol.api.dto.result.fromprotocol.Result;
+import de.adorsys.opba.protocol.api.dto.result.fromprotocol.ok.SuccessResult;
+import de.adorsys.opba.protocol.api.pis.GetPaymentStatusState;
+import de.adorsys.opba.protocol.api.services.scoped.consent.ProtocolFacingConsent;
+import de.adorsys.opba.protocol.xs2a.context.pis.Xs2aPisContext;
+import de.adorsys.opba.protocol.xs2a.entrypoint.ExtendWithServiceContext;
+import de.adorsys.opba.protocol.xs2a.entrypoint.helpers.UuidMapper;
+import de.adorsys.opba.protocol.xs2a.service.dto.ValidatedQueryHeaders;
+import de.adorsys.opba.protocol.xs2a.service.mapper.QueryHeadersMapperTemplate;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.dto.DtoMapper;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.dto.payment.PaymentStateHeaders;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.dto.payment.PaymentStateParameters;
+import de.adorsys.xs2a.adapter.service.PaymentInitiationService;
+import de.adorsys.xs2a.adapter.service.Response;
+import de.adorsys.xs2a.adapter.service.model.PaymentInitiationStatus;
+import lombok.RequiredArgsConstructor;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.springframework.stereotype.Service;
+
+import java.util.concurrent.CompletableFuture;
+
+import static de.adorsys.opba.protocol.xs2a.constant.GlobalConst.SPRING_KEYWORD;
+import static de.adorsys.opba.protocol.xs2a.constant.GlobalConst.XS2A_MAPPERS_PACKAGE;
+
+/**
+ * Entry point to get payment status. BPMN engine and process is not touched.
+ */
+@Service("xs2aGetPaymentStatusState")
+@RequiredArgsConstructor
+public class Xs2aGetPaymentStatusEntrypoint implements GetPaymentStatusState {
+    public static final String SEPA_CREDIT_TRANSFERS = "sepa-credit-transfers";
```

Comment:
This constant is duplicate with Xs2aGetPaymentInfoEntrypoint, please extract it into single Const class

Reply:
maybe better take it from request params?